src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -439,4 +439,7 @@
   <data name="ComplexTypeModelBinder_NoParameterlessConstructor_ForParameter" xml:space="preserve">
     <value>Could not create an instance of type '{0}'. Model bound complex types must not be abstract or value types and must have a parameterless constructor. Alternatively, give the '{1}' parameter a non-null default value.</value>
   </data>
+  <data name="ApiController_MultipleBodyParametersFound" xml:space="preserve">
+    <value>Action '{0}' has more than one parameter that were specified or inferred as bound from request body. Only one parameter per action may be bound from body. Inspect the following parameters, and use '{1}' to specify query string bound, '{2}' to specify route bound, and '{3}' for parameters to be bound from body:</value>
```

Comment:
"that *was*" (that phrase is about each parameter -- one at a time).

Not a big fan of "query string bound" or "route bound". Suggest using "bound from X" throughout.